cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/queries.spec.js

```diff
@@ -88,21 +104,15 @@ describe.skip('query enhancement queries', { scrollBehavior: false }, () => {
 
       // default PPL query should be set
       cy.waitForLoader(true);
-      cy.getElementByTestId(`osdQueryEditor__multiLine`).contains(`source = timestamp-*`);
+      cy.getElementByTestId(`osdQueryEditor__multiLine`).contains(
+        `source = data_logs_small_time_1*`
+      );
       cy.waitForSearch();
       cy.getElementByTestId(`queryResultCompleteMsg`).should('be.visible');
-      cy.get('[class="euiText euiText--small"]').then((text) => cy.log(text));
-      cy.verifyHitCount(4);
```

Comment:
Why do we delete verifying the hit count? If we delete verifying hit count, how are we ensuring the query is persisted across refresh? 

Feel like `cy.getElementByTestId(`queryResultCompleteMsg`).should('be.visible');` only does not provide valid proof

Reply:
sure. I added cy.verifyHitCount('10,000');